servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1071,6 +1104,17 @@ public String getUfsAddress() {
     return mWhitelist.getList();
   }
 
+  public List<Long> getLostFiles() {
+    Set<Long> lostFiles = Sets.newHashSet();
+    for (long blockId : mBlockMaster.getLostBlocks()) {
+      // the file id is the container id of the block id
+      long containerId = BlockId.getContainerId(blockId);
+      long fileId = InodeFile.toFileId(containerId);
+      lostFiles.add(fileId);
+    }
+    return new ArrayList<Long>(lostFiles);
+  }
+
   // TODO(gene): The following methods are for lineage, which is not fully functional yet.
```

Comment:
It seems strange to have both lineage implementations co-exist (for however brief period of time it is). It makes sense to do the cleanup in this PR (or before this PR).